.github/reviewers.txt

```diff
@@ -0,0 +1,5 @@
+# List of reviewers for auto-assignment of reviews.
+caseyclements
+blink1073
+Jibola
+NoahStapp
```

Comment:
Should shane be here as well?

Reply:
No, he's already exempt from the existing auto-assign.